src/runtime/styled.js

```diff
@@ -34,8 +34,9 @@ export function styled(
     if (hasModifiers) {
       Object.keys(props).forEach(propName => {
         const propValue = props[propName];
+        const propType = typeof propValue;
 
-        if (typeof propValue === 'boolean') {
+        if (propType === 'boolean' || propType === 'undefined') {
```

Comment:
maybe:

```js
typeof propValue === 'boolean' || propValue == null
```

?

seems harmless to make this work with `null` as well as `undefined`.

unfortunately this won't quite work for the string/number case, but this would be an incremental improvement

Reply:
Done https://github.com/4Catalyzer/astroturf/pull/57/commits/a9156387cff1b94d9f00a6bdfabb12c47ab127ec